src/documents/Sonification.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Sonification
+subheader:
+layout: default
+category: Methods
+---
+
+## What is Sonification?
+
+Sonification is a general term for conveying information or data through non-speech audio &mdash; compare to the much better-known term "visualization". Many of us are familiar with simple sonification as an aspect of the user interface of computers or other digital devices, such as alert sounds when receiving new email.
```

Comment:
Lead with the examples first since it's immediately more understandable. Rephrase to: "Alert sounds when receiving a new email is a simple sonification commonly used in user interfaces for computers and other devices."